app/Http/Controllers/NewsItemController.php

```diff
@@ -44,10 +46,22 @@ public function store(Request $request){
 
         //save image
         if($request->hasFile('thumbnail')){
-            $name  = $newsItem->id . '-thumbnail.' . $request->thumbnail->extension();
-            $request->file('thumbnail')->storeAs('newsItems', $name,'public');
-            $newsItem->image_url = 'newsItems/' .  $name;
+            //larger header image for detail view
+            $name_header = $newsItem->id . '-header.' . $request->thumbnail->extension();
+            $request->file('thumbnail')->storeAs('newsItems', $name_header,'public');
+            $newsItem->image_url = 'newsItems/' .  $name_header;
+
+            //smaller thumbnail image for list view
+            $name_thumbnail = $newsItem->id . '-header.' . $request->thumbnail->extension();
+            $request->file('thumbnail')->storeAs('newsItems', $name_thumbnail,'public');
+            $newsItem->thumbnail_url = 'newsItems/' .  $name_header;
             $newsItem->save();
+
+            //resize both images
+            $header_path = "../storage/app/public/" . $newsItem->image_url;
```

Comment:
Je kan hier beter de methode `storage_path()` gebruiken om het path the bepalen. 
https://laravel.com/docs/5.1/helpers#method-storage-path